Changelog.md

```diff
@@ -6,11 +6,16 @@
 
 [0.12.0.0]: https://github.com/haskell/bytestring/compare/0.11.3.0...0.12.0.0
 
-[0.11.3.0] — Unreleased
-
-* merge `shortbytestring` package back into `bytestring` wrt [#444](https://github.com/haskell/bytestring/issues/444),
-  adding lots of additional API:
-    - [Add `all`, `any`, `append`, `break`, `breakEnd`, `breakSubstring`, `concat`, `cons`, `count`, `drop`, `dropEnd`, `dropWhile`, `dropWhileEnd`, `elem`, `elemIndex`, `elemIndices`, `filter`, `find`, `findIndex`, `findIndices`, `foldl'`, `foldl`, `foldl1'`, `foldl1`, `foldr'`, `foldr`, `foldr1'`, `foldr1`, `head`, `init`, `intercalate`, `isInfixOf`, `isPrefixOf`, `isSuffixOf`, `last`, `map`, `partition`, `replicate`, `reverse`, `singleton`, `snoc`, `span`, `spanEnd`, `split`, `splitAt`, `splitWith`, `stripPrefix`, `stripSuffix`, `tail`, `take`, `takeEnd`, `takeWhile`, `takeWhileEnd`, `uncons`, `unfoldr`, `unfoldrN`, `unsnoc`](https://github.com/haskell/bytestring/pull/471)
+[0.11.3.0] — February 2022
+
+* [Merge `shortbytestring` package back into `bytestring`](https://github.com/haskell/bytestring/pull/471)
```

Comment:
I think this item is a bit confusing because

1. `shortbytestring` is not a well-known package.
2. It's unclear what "merging back" is supposed to mean. It seems to suggest that `shortbytestring` is/was a fork of `bytestring`.

```suggestion
* [Enhance `ShortByteString` API](https://github.com/haskell/bytestring/pull/471)
```

Reply:
> shortbytestring is not a well-known package.

:sob: